src/Client/Web/Bit.Client.Web.BlazorUI/Components/Notifications/BitMessageBar.razor.cs

```diff
@@ -1,88 +1,70 @@
-﻿using Microsoft.AspNetCore.Components;
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
 using Microsoft.AspNetCore.Components.Web;
 
 namespace Bit.Client.Web.BlazorUI
 {
     public partial class BitMessageBar
     {
-        private MessageBarStyle messageBarStyle = MessageBarStyle.Default;
+        protected override string RootElementClass => "bit-msg-bar";
 
-        [Parameter] public RenderFragment? ChildContent { get; set; }
+        [Parameter]
+        public bool IsMultiline { get; set; } = true;
 
-        [Parameter] public string? MessageBarIconName { get; set; }
+        [Parameter]
+        public BitMessageBarType MessageBarType { get; set; } = BitMessageBarType.Info;
```

Comment:
since this parameter affects the component's class builder, it needs to have a setter in order to reset the builder.

Reply:
Changed all properties according to our convention